alpha/apps/kaltura/lib/model/objectfilters/AdvancedSearchFilterMatchAttributeCondition.class.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model.filters.advanced
+ */ 
+class AdvancedSearchFilterMatchAttributeCondition extends AdvancedSearchFilterMatchCondition
+{
+	/**
+	 * Adds conditions, matches and where clauses to the query
+	 * @param IKalturaDbQuery $query
+	 */
+	public function applyCondition(IKalturaDbQuery $query)
+	{
+		if (!$query instanceof IKalturaIndexQuery)
+			return;
+
+		$matchText = '"'.SphinxUtils::escapeString($this->value).'"';
```

Comment:
Roman,
SphinxUtils is a plugin class.
Please consider moving the entire match functionality into search plugin, or, at least, generalize SphinxUtils::escapeString somehow.